integration/src/test/scala/ammonite/integration/LineNumberTests.scala

```diff
@@ -13,36 +14,48 @@ object LineNumberTests extends TestSuite{
   val tests = this{
 
     def checkErrorMessage(file: RelPath, expected: String) = {
-      val e = intercept[ShelloutException]{ 
+      val e = intercept[ShelloutException]{
         exec(file)
-      }.result.err.string.replace("\r", "").replace("\n", System.lineSeparator())
-      assert(e.contains(expected.replace("\n", System.lineSeparator())))
+      }.result.err.string
+      assert(e.contains(expected.replace("\n", Util.newLine)))
     }
 
-    'errorTest - checkErrorMessage(
-      file = 'lineNumbers/"ErrorLineNumberTest.sc",
-      expected =
-        """Syntax Error: ("}" | `case`):5:24 ...")\n  }\n\n  d"
-          |    printlnqs(unsorted))
-          |                       ^""".stripMargin
-    )
+    'errorTest - {
+      if(!Util.windowsPlatform) {
```

Comment:
Can you add a comment saying why these are skipped? I know it's probably something to do with fastparse error messages, as we discussed, but we should write it down so 6 months later we won't forget it when reading the code